src/ensembl/production/metadata/admin/admin.py

```diff
@@ -166,12 +166,16 @@ class OrganismGroupMemberInLine(MetadataInline, admin.StackedInline):
 @admin.register(Organism)
 class OrganismAdmin(AdminMetadata, admin.ModelAdmin):
     # assemblies
+    fields = (
+        'display_name', 'organism_uuid', 'genome_releases', 'organism_assemblies', 'strain', 'scientific_name', 'ensembl_name',
+        'scientific_parlance_name',
+        'url_name', 'taxonomy_id', 'species_taxonomy_id',)
     list_display = (
-        'display_name', 'genome_releases', 'organism_assemblies', 'strain', 'scientific_name', 'ensembl_name',
+        'display_name', 'organism_uuid', 'genome_releases', 'organism_assemblies', 'strain', 'scientific_name', 'ensembl_name',
         'scientific_parlance_name',
-        'url_name', 'display_name', 'taxonomy_id', 'species_taxonomy_id',)
+        'url_name', 'taxonomy_id', 'species_taxonomy_id',)
     list_filter = (MetadataReleaseFilter,)
-    search_fields = ('ensembl_name', 'assemblies__accession')
+    search_fields = ('ensembl_name', 'assemblies__accession', 'organism_uuid')
```

Comment:
Same here, can you add `ensembl_name` and `organism_uuid` to the readonly fields?